package.json

```diff
@@ -13,7 +13,7 @@
     "preview": "vite preview",
     "test:unit": "vitest",
     "test:e2e": "behave features",
-    "build-only": "vite build",
+    "build-only": "vite build && node scripts/version.js",
```

Comment:
Since this fails during build in OpenShift, maybe run this during `dev`

```diff
-    "dev": "vite",
+    "dev": "node scripts/version.js. && vite",
```